### PR TITLE
Risk Acceptance: Make API set/unset risk acceptance status

### PR DESCRIPTION
**Pull request by Maffooch**
When creating/updating/deleting a risk acceptance, the status of the finding is not set/unset even though the finding is added to the risk acceptance object.

[sc-6143]

### CODE REVIEW

**Review by blakeaowens — APPROVED** [review]

File: dojo/api_v2/serializers.py [hunk 1]
```diff
@@ -1485,6 +1509,12 @@ def get_engagement(self, obj):
         )
 
     def validate(self, data):
+        def validate_findings_have_same_engagement(finding_objects: List[Finding]):
+            engagements = finding_objects.values_list('test__engagement__id', flat=True).distinct().count()
+            if engagements > 1:
+                msg = "You are not permitted to add findings to a distinct engagement"
```
Comment (cneill) [inline dojo/api_v2/serializers.py:1515]: I think something like this better matches the intent here?

```suggestion
                msg = "You are not permitted to add findings from multiple engagements"
```

**Review by grendel513 — APPROVED** [review]

**Review by cneill — COMMENTED** [review]
Just a minor comment/question on this exception message, otherwise looks good

**Comment by Maffooch** [comment]
Tests have gotten stuck. Closing and reopening to trigger again

**Comment by Maffooch** [comment]
The settings.dist.py SHA was messed up

**Review by cneill — APPROVED** [review]

**Pull request merged**